app/scripts/lib/fxa-client.js

```diff
@@ -132,6 +132,8 @@ define(function (require, exports, module) {
       }
 
       var updatedSessionData = {
+        challengeMethod: accountData.challengeMethod,
```

Comment:
Fair enough, although I don't see the distinction between them.

I still think it would be useful for us to have the conversation anyway though. For instance, I'd like to discuss whether [the database PR](https://github.com/mozilla/fxa-auth-db-mysql/pull/132) should instead contain `unconfirmedTokens`, `tokenConfirmationId` and `/token/:tokenConfirmationId/confirm`.